build.gradle

```diff
@@ -210,6 +210,27 @@ processTestResources {
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
+def createSymlinks(archivePath, symlinkLocation) {
+    exec {
+        commandLine 'ln', '-fs', archivePath, symlinkLocation
+        ignoreExitValue = false
+    }
+}
+
+// Suffix is what will be added to the symlink
+def createAllSymlinks(destinationDir, archivePath, suffix) {
+    def finalSuffix = "-" + suffix
```

Comment:
Is there a conditional assign in Groovy/Gradle? so that you can do these 4 lines in one?

```
def finalSuffix = suffix == "" ? "" : "-" + suffix;
```